packages/resource-deployment/scripts/install-parallel.sh

```diff
@@ -140,12 +140,14 @@ function install() {
         "${0%/*}/setup-cosmos-db.sh"
         "${0%/*}/create-vnet.sh"
         "${0%/*}/app-insights-create.sh"
+        "${0%/*}/create-container-registry.sh"
     )
     runCommandsWithoutSecretsInParallel parallelProcesses
 
     # The following scripts all depend on the result from the above scripts.
     # Additionally, these should run sequentially because of interdependence.
 
+    . "${0%/*}/push-container-to-registry.sh"
```

Comment:
are you trying to push image to the container registry in this shell script? If yes do you mind changing the name appropriately.